components/booking/pages/AvailabilityPage.tsx

```diff
@@ -126,7 +126,11 @@ const AvailabilityPage = ({ profile, eventType, workingHours }: Props) => {
                           .filter((user) => user.name !== profile.name)
                           .map((user) => ({
                             title: user.name,
-                            image: user.avatar || undefined,
+                            image:
+                              (process.env.NEXT_PUBLIC_APP_URL || process.env.BASE_URL) +
+                              "/" +
+                              user.username +
+                              "/avatar.png",
```

Comment:
`process.env.BASE_URL` will always be undefined in the frontend.

```suggestion
                            image: `${process.env.NEXT_PUBLIC_APP_URL}/${user.username}/avatar.png`,
```